starter-code/pizza.js

```diff
@@ -1 +1,63 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function(){
+
+    // remove specific class names
+    $(".crust").removeClass("crust-gluten-free");
+    $(".sauce").removeClass("sauce-white");
+
+    // remove active for crust and sauce
+    $(".btn-crust, .btn-sauce").removeClass("active");
+
+    //hide sauce and crust price
+    $(".crust-price, .sauce-price").hide();
+
+    //calculate price onload and below onclick too
+    getPrice();
+
+    // calling 1 function 3 times so don't repeat code, but the second parameter is weird
+    ingredientsToggle(".btn-pepperonni", ".pep", ".pep-price");
+    ingredientsToggle(".btn-mushrooms", ".mushroom", ".mushroom-price");
+    ingredientsToggle(".btn-green-peppers", ".green-pepper", ".pepper-price");
+    //seperate function for crust and sauce so I can toggle the class name
+    ingredientsToggleWithClass(".btn-sauce", ".sauce", ".sauce-price", "sauce-white");
+    ingredientsToggleWithClass(".btn-crust", ".crust", ".crust-price", "crust-gluten-free");
+
```

Comment:
Por poneros una pega, os diría que hay demasiadas cosas dentro de la función `ready`; es mejor si ponéis varias de ellas fuera